system-int-test/test/cmr/system_int_test/search/granule_counts_search_test.clj

```diff
@@ -712,3 +712,48 @@
                                (search/find-refs :collection {:page-size 20
                                                               :sort-key ["-has_granules_or_opensearch"
                                                                          "revision-date"]}))))))
+
+(deftest collection-no-granule-count-test
+  (let [;; Create collections
+        ;; whole world, no temporal, and science keywords
+        coll1 (make-coll 1 m/whole-world nil)
+        ;; just like coll1
+        coll2 (make-coll 2 m/whole-world nil)]
+
+    ;; Ingest some granules for coll1 first, and leave coll2 for later
+    ;; Coll1 granules
+    (make-gran coll1 (p/point 0 0) nil)
+    (make-gran coll1 (p/point 0 90) nil)
+
+    (index/wait-until-indexed)
+
+    (testing "granule counts"
+      (testing "granule counts for all collections"
+        (let [refs (search/find-refs :collection {:include-granule-counts true})]
+          (is (gran-counts/granule-counts-match? :xml {coll1 2 coll2 0} refs)))))
```

Comment:
Since the granules are ingested directly before the tests, this check on granule counts is unnecessary, can be removed